tests/Command/DebugMappingCommandTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\Tests\Command;
+
+use League\Tactician\Bundle\Command\DebugMappingCommand;
+use League\Tactician\Bundle\DependencyInjection\HandlerMapping\Routing;
+use League\Tactician\Bundle\DependencyInjection\RoutingDebugReport;
+use League\Tactician\Bundle\Tests\Fake\FakeCommand;
+use League\Tactician\Bundle\Tests\Fake\OtherFakeCommand;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DebugMappingCommandTest extends TestCase
+{
+    public function testItShouldExecuteSuccessfully()
+    {
+        // GIVEN
+        $buses = [
+            'default',
+            'foo',
+            'bar'
+        ];
+
+        $report = $this->prepareReport($buses);
+        $command = new DebugMappingCommand($report);
+        $tester = new CommandTester($command);
+
+        // WHEN
+        $tester->execute([]);
+        $output = $tester->getDisplay();
+
+        // THEN
+        foreach ($buses as $bus) {
+            self::assertContains("Bus: $bus", $output);
+        }
+    }
+
+    /**
+     * @param $buses
+     * @return RoutingDebugReport
+     */
+    private function prepareReport(array $buses): RoutingDebugReport
+    {
+        $routing = new Routing($buses);
+
+        $routing->routeToBus('bar', FakeCommand::class, 'fake.handler');
+        $routing->routeToBus('foo', OtherFakeCommand::class, 'other_fake.handler');
+
+        $mappings = [];
+        foreach ($buses as $bus) {
+            $mappings[$bus] = $routing->commandToServiceMapping($bus);
+        }
+
+        return new RoutingDebugReport($mappings);
```

Comment:
Same as the comment below, you can simplify this test a lot by just passing a mapping array you hardcode here, rather than creating a routing object. If we ever change the format, we'll update the tests too, no big deal. It's a good way to know the two are coupled. :)